server/game/cards/07-WotW/SaadiyahAlMozedu.js

```diff
@@ -0,0 +1,30 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { Locations, CardTypes } = require('../../Constants');
+
+class SaadiyahAlMozedu extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Flip province facedown',
+            cost: AbilityDsl.costs.discardCard({
+                location: Locations.Hand,
+                targets: true
```

Comment:
Don't think you want `targets: true` because the card does not use the word "choose" with regard to the discard